### PR TITLE
Updates install instructions in README

### PR DESCRIPTION
**Pull request by liminalisht**
This PR:
- updates the readme to include more detailed instructions on compiling and running w/ `stack`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,15 +72,38 @@ $ nix-build
...
+You'll need a runtime dependency: goto https://www.oracle.com/database/technologies/instant-client/macos-intel-x86-downloads.html#ic_osx_inst and follow the instant client installation instructions.
+
+Then link a dynamic lib from the instant client to a location on your host where it can be found:
+```
+ln -s ~/Downloads/instantclient_19_8/libclntsh.dylib /usr/local/lib/
 ```
 
+Run docker-compose up and tests as so:
+``` bash
+docker-compose up
```
Comment (dmjio) [inline README.md]: Should this be `docker-compose up -d` ?

File: README.md [hunk 2]
```diff
@@ -68,15 +72,38 @@ $ nix-build
...
-## Test
+First install `odpi` (e.g. on MacOS):
+``` bash
+brew install odpi
+```
 
+This should suffice to permit you to build:
 ```bash
-$ docker-compose up
-$ nix-build && ./result/bin/example
```
Comment (dmjio) [inline README.md:80]: Can we keep the `nix` instructions in here too

File: README.md [hunk 3]
```diff
@@ -68,15 +72,38 @@ $ nix-build
...
 
-## Test
+First install `odpi` (e.g. on MacOS):
+``` bash
+brew install odpi
+```
 
+This should suffice to permit you to build:
 ```bash
-$ docker-compose up
```
Comment (dmjio) [inline README.md:79]: `-d` here too I think

File: README.md [hunk 4]
```diff
@@ -68,15 +72,38 @@ $ nix-build
 $ nix-shell --run 'cabal build'
```
Comment (dmjio) [inline README.md:72]: Could we also put `nix-build && ./result/bin/example` here too, just in case

File: README.md (hunk 2 above)
Comment (dmjio) [reply README.md:80]: Commented again above

File: README.md (hunk 4 above)
Comment (liminalisht) [reply README.md:72]: it is here, just two lines below

File: README.md (hunk 3 above)
Comment (liminalisht) [reply README.md:79]: 👍

**Review by dmjio — CHANGES_REQUESTED** [review]
Looks good, I'd ask to add:
  -  `-d` to the `docker-compose` invocation
  - Keep the `nix-build && ./result/bin/example`

**Comment by liminalisht** [comment]
I added the -d flag.

I didn't remove the `nix-build && ./result/bin/example` example, just moved it.

**Review by dmjio — APPROVED** [review]
👍🏼

**Pull request merged**